multiqc/utils/search_patterns.yaml

```diff
@@ -281,7 +281,7 @@ diamond:
 dragen/vc_metrics:
   fn: "*.vc_metrics.csv"
 dragen/ploidy_estimation_metrics:
-  fn: "*.ploidy_estimation_metrics.csv"
+  fn_re: '.*\.ploidy_estimation_metrics\.csv'
```

Comment:
l guess so. l never investigated how "fn" works, just presumed that it is some wrapper and decided to take a regex, because it was easier for me to work with.

Reply:
Yes, it's identical. Just fewer backslashes required 😅 (and maybe faster? but I might be making that up). [Syntax here](https://en.wikipedia.org/wiki/Glob_(programming)#Syntax).

I'd prefer to keep glob unless regex is needed, it's simpler and easier to read / less likely to have unintended consequences.

Reply:
Thanks for the link. The glob is restored now.

Reply:
> maybe faster
It is faster!

Reply:
> l never investigated how "fn" works, just presumed that it is some wrapper and decided to take a regex, because it was easier for me to work with.

It's just a unix [glob pattern](https://en.wikipedia.org/wiki/Glob_(programming)). The same you use to select files in the bash shell.